sites/friday/src/projects/region-migration.md

```diff
@@ -0,0 +1,149 @@
+Otherwise, use the desired plan size from the start.
+
+## 3. Add code and environments
+
+{{< codetabs >}}
+
++++
+title=Without a source integration
++++
+
+1. Clone your existing project with Git.
+2. In the new clone, add a remote for the project:
+
+   ```bash
+   {{% vendor/cli %}} project:set-remote
+   ```
+
+   Select your newly created blank project.
+
+3. Push the code for your production branch:
+
+   ```bash
+   {{% vendor/cli %}} push --target {{% variable "PRODUCTION_BRANCH_NAME" %}}
+   ```
+
+4. (Optional) Checkout other branches and then push their code:
+
+   ```bash
+   {{% vendor/cli %}} push --activate --target {{% variable "BRANCH_NAME" %}} --parent {{% variable "PRODUCTION_BRANCH_NAME" %}}
+   ```
+
+<--->
+
++++
+title=With a source integration
++++
+
+For a [source integration](../integrations/source/_index.md) with GitHub, BitBucket, or GitLab,
+add the integration to your new project.
+Your new project then mirrors the configured repository automatically.
+
+{{< /codetabs >}}
+
+## 4. Copy files
+
+If you have files in a mount, first download them:
+
+```bash
+{{% vendor/cli %}} mount:download
+```
+
+Then upload them to your new project:
+
+```bash
+{{% vendor/cli %}} mount:upload
+```
+
+See more options on [how to export files](/learn/tutorials/exporting.md)
+and [how to import files](/learn/tutorials/migrating/from-psh.md#6-import-data).
+
+## 5. Copy data from services
+
+For services with generated data such as Solr and Redis, you don't need to copy data directly.
+Just rebuild the data in the new project.
+
+To download data from persistent services such as databases,
+see how to export and then import data for each service:
+
+- [InfluxDB](../add-services/influxdb.md#export-data)
+- [MongoDB](../add-services/mongodb.md#exporting-data)
+- [MariaDB/MySQL](../add-services/mysql/_index.md#exporting-data)
+- [PostgreSQL](../add-services/postgresql.md#exporting-data)
+
+## 6. Migrate variables and project settings
+
+Make sure anything else connected to your old project is moved to your new project:
+
+- If you have project or environment variables defined on your old project, add them to your new project.
+  Get a list of all variables set outside of code by running `{{% vendor/cli %}} variables`.
+- Add any users to your new project that you want to continue to have access.
+- Add any existing [integrations](../integrations/_index.md).
+
+## 7. Test the site
+
+Verify that the new site is working as desired before continuing.
+You can leave the two projects running for as long as you need.
+After you have finished all your testing, sync all your data (code, files, database) for the last time.
+
+## 8. Switch to the new site
+
+Now that you know the new project works, switch public traffic to that site:
+
+1. Make sure your new project has the right plan size.
+2. If possible, put your site into read-only mode or maintenance mode.
```

Comment:
as for custom SSL certificates, add `(Optional)` at the beginning?

Reply:
It does, unless I'm not understanding? 
`4. (Optional) Add any custom SSL certificates you have.`